Source/stylecop.json

```diff
@@ -8,7 +8,7 @@
   "$schema": "https://raw.githubusercontent.com/DotNetAnalyzers/StyleCopAnalyzers/master/StyleCop.Analyzers/StyleCop.Analyzers/Settings/stylecop.schema.json",
   "settings": {
     "documentationRules": {
-      "companyName": "Microsoft"
```

Comment:
I don't think this is the right place to change it. You're just changing the Stylecop rule not the actual text.